### ISSUE TITLE
Add config path output logging on nodeos

### ISSUE THREAD

**Issue by bensig** [EOSIO/eos#8221]
After upgrading to eosio 1.8.6 recently, I was unable to run my local environment. 

I upgraded with `brew upgrade eosio` 

Standalone node would not run due to the error: 

`error 2019-11-14T02:31:22.199 thread-0  main.cpp:120                  main                 ] Throw location unknown (consider using BOOST_THROW_EXCEPTION)
Dynamic exception type: boost::wrapexcept<boost::program_options::unknown_option>
std::exception::what: unrecognised option 'bnet-endpoint'
`
Only, the config file I was using did not have bnet specified... I also was unable to downgrade via brew. Perhaps it would be a good idea to store some additional versions of EOSIO and EOSIO.CDT in the cellar... 

Through a lot trials and searching, I finally found the default config location (which is pretty obscure) at `~/Library/Application\ Support/eosio/nodeos/`

There was a VERY old config file there from Nov 2018... 

I would like to suggest that a line be added to nodeos output which shows the configuration file path being used.

**Comment by bensig** [comment]
TLDR; EOSIO default config location on OS X is too obscure to locate, a simple output of the config.ini path on nodeos logs would help immensely with making nodeos more operator friendly on ANY system.

**Comment by bensig** [comment]
Update: this is not just an issue on OS X, but in other systems. New developers are having trouble running the EOSIO binaries.

**Comment by jgiszczak** [comment]
`nodeos` since 2 Nov 2018 has output the configuration file used during startup, e.g.
```
info  2019-11-14T21:03:00.775 nodeos    main.cpp:94                   main                 ] nodeos using configuration file /home/user/.local/share/eosio/nodeos/config/config.ini
```
This has been available since release version 1.4.3.
